docs/next/modules/en/pages/tutorials/quickstart.adoc

```diff
@@ -44,13 +44,13 @@ If you're customizing the installation parameters, please make sure that you are
 | Name | Version 
 
 | Cluster API Operator
-| `v0.17.0`
+| `v0.18.0`
 
 | Cluster API
-| `v1.9.5`
+| `v1.9.6`
 
 | Cluster API Provider RKE2
-| `v0.12.0`
+| `v0.13.0`
```

Comment:
note: this assumes the new CAPRKE2 minor release (not available yet) will be used in Turtles.

Reply:
release v0.13.0 is out.